package.json

```diff
@@ -9,15 +9,17 @@
   "scripts": {
     "postinstall": "scripts/react-native-web.sh && cd desktop && npm install",
     "clean": "react-native clean-project-auto",
-    "android": "npm run check-metro-bundler-port && react-native run-android",
-    "ios": "npm run check-metro-bundler-port && react-native run-ios",
+    "android": "npm run check-metro-bundler-port && scripts/dev.sh && react-native run-android",
+    "ios": "npm run check-metro-bundler-port && scripts/dev.sh && react-native run-ios",
     "ipad": "npm run check-metro-bundler-port && react-native run-ios --simulator=\"iPad Pro (12.9-inch) (4th generation)\"",
     "ipad-sm": "npm run check-metro-bundler-port && react-native run-ios --simulator=\"iPad Pro (9.7-inch)\"",
     "start": "react-native start",
-    "web": "node web/proxy.js & webpack-dev-server --open --config config/webpack/webpack.dev.js",
+    "web": "concurrently npm:web-proxy npm:web-server",
+    "web-server": "scripts/dev.sh && webpack-dev-server --open --config config/webpack/webpack.dev.js",
+    "web-proxy": "node web/proxy.js",
```

Comment:
```
concurrently npm:web-proxy npm:web-server
```

It's the same as 
```
npm run web-proxy & npm run web-server
```

but it's cross platform compatible and output is prefixed from the script it comes from 

```
[web-server] > new.expensify@1.1.57-3 web-server /Users/kidroca/Desktop/Freelance/Expensify.cash
[web-server] > scripts/dev.sh && webpack-dev-server --config config/webpack/webpack.dev.js
[web-server] 
[web-proxy] 
[web-proxy] > new.expensify@1.1.57-3 web-proxy /Users/kidroca/Desktop/Freelance/Expensify.cash
[web-proxy] > node web/proxy.js

```